OutOfSchool/OutOfSchool.AuthServer.Tests/EmailSender/EmailSenderJobTests.cs

```diff
@@ -86,7 +86,7 @@ public async Task Execute_WithExpiredEmail_ShouldNotSendEmail()
                 { EmailSenderStringConstants.Subject, "Test Email" },
                 { EmailSenderStringConstants.HtmlContent, Convert.ToBase64String(Encoding.ASCII.GetBytes("<html><body><h1>Hello</h1></body></html>")) },
                 { EmailSenderStringConstants.PlainContent, Convert.ToBase64String(Encoding.ASCII.GetBytes("Hello")) },
-                { EmailSenderStringConstants.ExpirationTime, DateTimeOffset.Now.AddMinutes(-10).ToString() }
+                { EmailSenderStringConstants.ExpirationTime, DateTimeOffset.Now.AddMinutes(-10).ToString("dd.MM.yyyy HH:mm:ss zzz") }
```

Comment:
(Code style, minor) This date format string literal might be a class-level constant. Just to minimize possible issues while editing, and (in case of differnt format used in some test) explicitely highlight that it is different.

Reply:
Fixed